framework/src/controller/validator/keywords/env/index.js

```diff
@@ -18,7 +18,23 @@ const compile = (schema, parentSchema) => {
 			  };
 
 	return function(data, dataPath, object, key) {
-		const variableValue = process.env[envVariable.name];
+		let variableValue = process.env[envVariable.name];
+
+		// Formatting logic to automatically format the CLI value to the expected type
+		// Only format if the variable exist
+		if (process.env[envVariable.name] !== undefined) {
```

Comment:
I'd prefer to use `variableValue` instead of `process.env[envVariable.name]`, since it's easier to read.

Reply:
Ah good catch!